public/js/product/components/ProductPriceInfo.css

```
@@ -0,0 +1,193 @@
+    font-size: 11px;
+    font-weight: 600;
+    text-transform: uppercase;
+}
+
+.discount-summary {
+    margin-top: 10px;
+    padding-top: 10px;
+    border-top: 1px solid #e8f5e8;
+    background: #f0f8f0;
+    margin: 10px -20px -20px -20px;
+    padding: 15px 20px;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)